### PR TITLE
minor svcstats & sanity fix

### PR DESCRIPTION
**Pull request by shaleman**
- service stats fix
- ssh slowness fix for ubuntu

### CODE REVIEW

File: netmaster/objApi/objapi_test.go [hunk 1]
```diff
@@ -1002,6 +1002,10 @@ func TestNetworkAddDelete(t *testing.T) {
 	verifyNetworkState(t, "default", "contiv", "data", "vxlan", "10.1.1.1", "10.1.1.254", 16, 1, 1, "", "", 0)
 	checkDeleteNetwork(t, false, "default", "contiv")
 
+	// vxlan network with VNI higher than 4K
+	checkCreateNetwork(t, false, "default", "contiv", "", "vxlan", "10.1.1.1/24", "10.1.1.254", 8000, "", "")
+	checkDeleteNetwork(t, false, "default", "contiv")
+
```
Comment (gaurav-dalvi) [inline netmaster/objApi/objapi_test.go]: Is this related to Issue : https://github.com/contiv/netplugin/issues/508

File: Vagrantfile [hunk 2]
```diff
@@ -67,6 +67,10 @@ sudo systemctl start docker
 rm /etc/docker/key.json
 (service docker restart) || exit 1
 
+# fix SSH slowness
+sed -i -e 's/UsePAM yes/UsePAM no/g' /etc/ssh/sshd_config
+(service sshd restart) || exit 1
+
```
Comment (gaurav-dalvi) [inline Vagrantfile]: are we going to do this for all OSes ? or just for ubuntu ?

**Comment by gaurav-dalvi** [comment]
apart from some minor comments, LGTM

File: netmaster/objApi/objapi_test.go (hunk 1 above)
Comment (shaleman) [inline netmaster/objApi/objapi_test.go]: yes. it fixes #508

File: Vagrantfile (hunk 2 above)
Comment (shaleman) [inline Vagrantfile]: yeah, for all OSes. We don't see any issues with Centos with this change.

**Comment by gaurav-dalvi** [comment]
@jojimt @abhinandanpb : Do we still need this ? If yes then we need to submit new PR for this.

File: netmaster/objApi/objapi_test.go (hunk 1 above)
Comment (mhrobinson) [inline netmaster/objApi/objapi_test.go]: It looks like this commit only tests for https://github.com/contiv/netplugin/issues/508 but doesn't fix it.  I'm not seeing the issue though, so probably some other commit fixed it?

**Pull request closed without merging**